pkg/controller/host/host_controller.go

```diff
@@ -208,7 +208,7 @@ func (r *ReconcileHost) buildInitialHostOpts(instance *starlingxv1.Host, profile
 
 func provisioningAllowed(objects []hosts.Host) bool {
 	for _, host := range objects {
-		if host.Hostname == hosts.Controller0 {
+		if host.Hostname == hosts.Controller0 || host.Hostname == "controller-1" {
```

Comment:
Is the plan to create a PR for gophercloud longer term?

Reply:
If anything, I'd rather drop the Controller0 constant from gophercloud and create constants locally within the DM repo, since it doesn't appear to be used anywhere in gophercloud

Reply:
I agree.  This is not an API constant but rather a well know resource of StarlingX and therefore doesn't belong in the gophercloud API.